plugins/modules/zos_volume_init.py

```diff
@@ -0,0 +1,242 @@
+  - "Almigdad Suliman (@Almigdad-Suliman)"
+  - "Nicholas Teves (@TODO)"
+  - "Nuoya Xie (@nxie13)"
+  - "Trevor Glassey (@tkglassey)"
+  - "Tyler Edwards (@TLEdwards-Git)"
+  - "Ketan Kelkar (@ketankelkar)"
+
+options:
+  volume_address:
+    description:
+      - 3 or 4 hexadecimal digit address of the volume to initialize.
+      - This could be a unit address or device number.
+    required: true
+    type: str
+  verify_existing_volid:
+    description:
+      - Verify that the provided volume serial matches the one found on existing volume/minidisk.
+      - Module fails if volser does not match.
+      - Note - This option is NOT a boolean, please leave it blank in order to skip the verification.
+    required: false
+    type: str
+  verify_offline:
+    description:
+      - Verify that the device is offline to all other systems.
+      - Beware, defaults set on target z/OS systems may override ICKDSF parameters.
+    type: bool
+    required: false
+    default: true
+  volid:
+    description:
+      - Specify the volume serial number to initialize the volume with.
+      - Expects 1-6 alphanumeric, national ($, \\#, \\@) or special characters.
+      - volid's specified with less than 6 characters are left justified and padded with blank chars (X'40').
+      - Characters are not validated so check with operating system guide for valid alphanumeric characters.
+      - Also referred to as volser or volume serial number.
+      - Default behavior in the case this parameter is not specified in the case of an existing device initialization is the reuse the existing volume serial.
+    required: false
+    type: str
+  vtoc_tracks:
+    description:
+      - The number of tracks to initialize the VTOC with.
+      - The VTOC will be placed at cylinder 0 head 1 for the number of tracks specified.
+      - Typically, ICKDSF will default the size to the number of tracks in a cylinder minus 1.
+      - For a 3390, the default is cylinder 0, track 1 for 14 tracks
+    required: false
+    type: int
+  index:
+    description:
+      - Create a VTOC index during volume initialization.
+      - The index size will be based on the size of the volume and the size of the VTOC that was created.
+      - The index will be placed on the volume after the VTOC.
+      - Set to false to not generate an index.
+    required: false
+    type: bool
+    default: true
+  sms_managed:
+    description:
+      - Assigned to be managed by Storage Management System (SMS).
+    type: bool
+    required: false
+    default: true
+  verify_no_data_sets_exist:
+    description:
+      - Verify if data sets other than the VTOC index data set and/or VVDS exist on the volume to be initialized.
+      - Beware that z/OS system defaults can override ICKDSF parameters.
+    required: false
+    type: bool
+    default: true
+"""
+EXAMPLES = r"""
+- name: Initialize target volume with all default options. Target volume address is '1234', set volume name to 'DEMO01'.
```

Comment:
are these not covered by sanity tests :(
maybe i have to generate docs and run on those generated files

Reply:
They should be covered under the Sanity tests, Ansible sometimes override the defaults and relaxes some rules, this could one of those that allow up to 120, i don't recall offhand. If it is passing that is fine, i tend to sent my margin at 80 just to be on the safe side. 

The doc generation catches some things , like syntax errors but I don't think the PEP8 rules are enforced